src/SignalR/clients/ts/webdriver-tap-runner/package.json

```diff
@@ -12,14 +12,14 @@
   "author": "",
   "license": "ISC",
   "dependencies": {
-    "chromedriver": "^91.0.0",
+    "chromedriver": "^93.0.0",
```

Comment:
Not sure what we want to do with the TS client on 2.1, we no longer ship it so shouldn't need to maintain it.